Dfe.Data.SearchPrototype/Search/Application/Adapters/ISearchServiceAdapter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Dfe.Data.SearchPrototype.Search.Domain.AggregateRoot;
+
+namespace Dfe.Data.SearchPrototype.Search.Application.Adapters
+{
```

Comment:
When you're talking about adaptors, that's in the context of ports and adaptors? So this is the port? I think it's worth discussing with Scott whether this code is meant as part-tutorial and if it is, these comments need to be really well thought-through (and maybe the word 'port' used, if that is the language that we're using)